core/block_svg.js

```diff
@@ -29,6 +29,7 @@ goog.provide('Blockly.BlockSvg');
 goog.require('Blockly.Block');
 goog.require('Blockly.blockAnimations');
 goog.require('Blockly.blockRendering.IPathObject');
+goog.require('Blockly.Comment');
```

Comment:
This sounds good to me, can we move ``
goog.require('Blockly.Warning');`` into block_svg as part of this change as well. 
Since you're making Icon only accept BlockSvg and Warning is a subclass of Icon.

Reply:
I moved warning as well as mutator.

Reply:
Note, Neil just removed this require as part of making it optional: https://github.com/google/blockly/pull/3053

Reply:
Yeah so you can remove all three from here.